errors/src/errors/parser/parser_errors.rs

```diff
@@ -422,4 +422,26 @@ create_messages!(
             ))
         },
     }
+
+    @formatted
+    could_not_lex_span {
+    args: (input: impl Display),
+    msg: format!("Could not lex the following content: `{input}`.\n"),
+    help: None,
+    }
+
+    /// For when the lexer encountered a bidi override character
```

Comment:
Do any of the existing errors need to be deprecated?

Reply:
I just marked as unused all the unused errors in this file, both newly unused ones and ones that were already not used.